### PR TITLE
fix: Lower Logback checkIncrement value

### PR DESCRIPTION
**Pull request by labrenbe**
# Description
Partially fixes https://github.com/stackabletech/issues/issues/574.

By default Logback checks the size of log files every 60s to determine if log rotation should be triggered. This can be too slow in NiFi with high load resulting in log files becoming larger than the provisioned `emptyDir` volume. Lowering the `checkIncrement` property allows for more timely file size checks.

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes

```[tasklist]
# Author
- [x] Changes are OpenShift compatible
- [x] CRD changes approved
- [x] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [x] Changelog updated
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

### CODE REVIEW

File: crates/stackable-operator/src/product_logging/framework.rs [hunk 1]
```diff
@@ -589,6 +589,7 @@ pub fn create_logback_config(
     </rollingPolicy>
     <triggeringPolicy class="ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy">
       <MaxFileSize>{max_log_file_size_in_mib}MB</MaxFileSize>
+      <checkIncrement>5s</checkIncrement>
```
Comment (siegfriedweber) [inline crates/stackable-operator/src/product_logging/framework.rs:592]: `5s` sets `checkIncrement` to 5 milliseconds.

The plural `s` is ignored in the pattern of the unit part:
https://github.com/qos-ch/logback/blob/v_1.5.7/logback-core/src/main/java/ch/qos/logback/core/util/Duration.java#L43

Therefore, this value has effectively no unit which defaults to milliseconds:
https://github.com/qos-ch/logback/blob/v_1.5.7/logback-core/src/main/java/ch/qos/logback/core/util/Duration.java#L96

The unit `second` must be used instead:

```suggestion
      <checkIncrement>5 seconds</checkIncrement>
```

**Review by siegfriedweber — CHANGES_REQUESTED** [review]
This change must be added to the changelog.

File: crates/stackable-operator/CHANGELOG.md [hunk 2]
```diff
@@ -39,6 +40,7 @@ All notable changes to this project will be documented in this file.
 [#840]: https://github.com/stackabletech/operator-rs/pull/840
 [#841]: https://github.com/stackabletech/operator-rs/pull/841
 [#843]: https://github.com/stackabletech/operator-rs/pull/843
+[#853]: https://github.com/stackabletech/operator-rs/pull/853
```
Comment (siegfriedweber) [inline crates/stackable-operator/CHANGELOG.md:43]: This line must be moved to the Unreleased section.

**Review by siegfriedweber — CHANGES_REQUESTED** [review]

File: crates/stackable-operator/CHANGELOG.md (hunk 2 above)
Comment (labrenbe) [reply crates/stackable-operator/CHANGELOG.md]: Thanks for spotting that

**Review by siegfriedweber — APPROVED** [review]
LGTM

**Pull request merged**